app/views/dashboard/protocols/show.html.haml

```diff
@@ -25,7 +25,7 @@
   = render 'dashboard/service_requests/service_requests', protocol: @protocol, super_user_orgs: @super_user_orgs, permission_to_edit: @permission_to_edit, user: @user, view_only: false, show_view_ssr_back: @show_view_ssr_back, statuses_hidden: %(first_draft)
   .additional-details-submissions-panel
     = render "additional_details/submissions/submissions_panel", protocol: @protocol, submissions: @submissions
-  .milestones-panel
+  .milestones-panel.spacing-above
```

Comment:
Instead of adding a class to do this, moving the `panel` and `panel-default` div to this would do the same thing, using the built-in bootstrap `panel-group` styling.